python/alibiexplainer/alibiexplainer/__main__.py

```diff
@@ -29,6 +30,9 @@
 def main():
     args, extra = parse_args(sys.argv[1:])
     # Pretrained Alibi explainer
+
+    nest_asyncio.apply()
```

Comment:
do we still need this line ?

Reply:
Yes. This is required without an upstream change to `alibi` to make it support an async `predictor_fn`.

Reply:
That line is what actually patches `asyncio` to be reentrant.